motoman_gp225_support/package.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>motoman_gp225_support</name>
+  <version>0.1.0</version>
+  <description>
+    <p>ROS-Industrial support for the Motoman GP225.</p>
+    <p>
+      This package contains configuration data, 3D models and launch files
+      for Motoman GP225 series of manipulators..
+    </p>
+    <p>
+      <b>Specifications</b>
+    </p>
+    <ul>
+      <li>GP225 - Default</li>
+    </ul>
+    <p>
+      Joint limits and maximum joint velocities are based on the information 
+      found in <em>YASKAWA MOTOMAN-GP225,GP180,GP180-120 INSTRUCTIONS (HW1484398)</em>
+      version <em>182739-1CD, rev 5</em>.
+    </p>
+    <p>
+      Before using any of the configuration files and / or meshes included
+      in this package, be sure to check they are correct for the particular
+      robot model and configuration you intend to use them with.
+    </p>
+    <p>
+      NOTE: the base model xacro macro has all joints oriented such that they
+      rotate around the Z+ axis. As a consequence, link frames in this model
+      do not follow REP-103 wrt frame orientation.
+    </p>
```

Comment:
This does not appear to be true for this package.

Copy-pasta?

Reply:
That is correct. It probably came from using another model as a base to build this one.